_topic_maps/_topic_map.yml

```diff
@@ -2806,8 +2806,10 @@ Topics:
   Topics:
   - Name: Installing distributed tracing
     File: distr-tracing-installing
-  - Name: Configuring distributed tracing
+  - Name: Configuring distributed tracing platform
```

Comment:
Sorry if I don't understanding the terminology enough to know, but should it be "Configuring the distributed tracing platform"?